tests/ttnn/unit_tests/gtests/ccl/test_fabric_edm_common.hpp

```diff
@@ -423,7 +424,8 @@ void generate_sender_worker_kernels(
         page_plus_header_size - PACKET_HEADER_SIZE_BYTES,
         worker_fabric_connection.num_buffers_per_channel,
         dest_is_dram,
-        std::holds_alternative<mcast_send>(mode) ? 1 : 0};
+        std::holds_alternative<mcast_send>(mode) ? 1 : 0,
+        scatter_write ? 1 : 0};
```

Comment:
```suggestion
        scatter_write};
```
don't need ternary. Bool will convert to 1 or 0 if casted to int